src/hb-ot-var-avar-table.hh

```diff
@@ -133,14 +168,27 @@ struct avar
     }
   }
 
+  int unmap_coord (unsigned int index, int coord) const
```

Comment:
Like `map_coords`, implement `unmap_coords` that does all coords in one go.  Otherwise it will become O(n^2).  It doesn't have to be.

Reply:
Done